docs/content/guide/dxos-ui/snippets/vite-config.ts

```diff
@@ -7,6 +7,7 @@ import { defineConfig } from 'vite';
 
 import { ThemePlugin } from '@dxos/react-ui-theme/plugin';
 
+// https://vitejs.dev/config
```

Comment:
I don't think this need to be included into the snippet for the docs

Reply:
i thought snippets were compiled to check they were legit?

Reply:
i've been fighting this for an hour:

```ts
      docs/content/guide/dxos-ui/snippets/vite-config.ts:13:5 - error TS2769: No overload matches this call.
        The last overload gave the following error.
          Type 'Plugin<any>' is not assignable to type 'PluginOption'
```

Reply:
they are, but they're also rendered on the docs site

Reply:
oh sorry, i thought you meant the import.

Reply:
NOTE: I had to update @types/node to "@types/node": "^22.7.7" to get the error to go away.

Reply:
^ FYI @dmaretskyi